pkgs/by-name/go/gotree/package.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "gotree";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "elbachir-one";
+    repo = "gt";
+    rev = "v${version}";
+    hash = "sha256-sM4iEsobN4ZLVW2Nk1/lP5utsu8qaKIytxrOPK3/2nY=";
+  };
+
+  goPackagePath = "github.com/elbachir-one/gt";
+
+  meta = {
+    description = "gt is a tree-like program written in Go with a lot of features.";
```

I'd like to suggest to change the `meta.description` as follows to be more inline with the recommendations in [`pkgs/README.md`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes)
```suggestion
    description = "Display a tree of files and directories";
```